packages/grafana-icons/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@grafana/saga-icons",
+  "version": "0.0.0",
```

I think the best way to iterate on this package is to make it private right now and get this merged in:

```suggestion
  "version": "0.0.0",
  "private": true,
```